addons/cargo/XEH_postInit.sqf

```diff
@@ -50,6 +50,7 @@
 
     _item hideObjectGlobal false;
     _item setPosASL (AGLtoASL _emptyPosAGL);
+    _item setDamage 0;
```

Comment:
We used this to prevent Vehicles from exploding on unloading.

Maybe the Vehicle should become invincible for X second to prevent instant explosions on unload?

Reply:
No, that would be weird.

Reply:
Do you have any smart suggestion to prevent Vehicles randomly exploding because a little stone get in their hitbox and physics goes crazy?

This is a serious problem on unloading Vehicles from Cargoinventory.

Reply:
So you clearly mean the item should be invulnerable, not the vehicle.

Reply:
If the item exploded anyway, setDamage will not revive it, and it will not stop the physics.